CHANGELOG

```diff
@@ -1,3 +1,6 @@
+2.0.4 2015-08-12
+- Improve pidfile management in upstart job
```

Comment:
Would it be clearer to say that we changed to use the new pidfile that carbon-cache uses?